examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -268,3 +271,14 @@ void emberAfWindowCoveringClusterInitCallback(chip::EndpointId endpoint)
     Clusters::WindowCovering::SetDefaultDelegate(endpoint, &sWindowCoveringManager);
     Clusters::WindowCovering::ConfigStatusUpdateFeatures(endpoint);
 }
+
+using namespace chip::app::Clusters::DiagnosticLogs;
+void emberAfDiagnosticLogsClusterInitCallback(chip::EndpointId endpoint)
+{
+    ChipLogProgress(NotSpecified, "SetDiagnosticLogsProviderDelegate");
```

Comment:
maybe we should just add a tracepoint instead - this seems not that useful except manual debugging that this method was exectued.